app/lib/l10n/app_en.arb

```diff
@@ -199,6 +199,8 @@
   "@convertingToActerSpace": {},
   "copyUsername": "Copy username",
   "@copyUsername": {},
+  "copyMessage": "Copy",
```

Comment:
```suggestion
  "copy": "Copy",
```

Better to make it generic as it can be useful at other place also

Reply:
I was considering that, but then thought that I could see the the case where this would be a different term than using the general one in other languages so added the specifier "Message" (as in "copy this message" - not as a generic term). I guess one can argue either way...

Reply:
No problem, understood your thoughts. We can keep this as it is.